pkgs/development/python-modules/secretstorage/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage
+, dbus-python, cryptography }:
+
+buildPythonPackage rec {
+  pname = "secretstorage";
+  version = "2.3.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "mitya57";
+    repo = "secretstorage";
+    rev = version;
+    sha256 = "1sjd2jjbxgkkxyrfwx89x0hsnn39w2cr2qkxbg1iz52znr4sqism";
+  };
+
+  postPatch = "patchShebangs .";
```

Comment:
why is this needed?

Reply:
this is a leftover from the derivation I took inspiration from. It should not be necessary.